code/src/sixsq/nuvla/server/resources/notification.clj

```diff
@@ -0,0 +1,120 @@
+(ns sixsq.nuvla.server.resources.notification
+  "
+Notification resource.
```

Comment:
You'll want to provide a better description before this is merged.

Reply:
Partially addressed in ff6f901